mail_message_icon/wizard/mail_resend_message.py

```diff
@@ -0,0 +1,49 @@
+from odoo import Command, api, fields, models
+
+
+class MailResendMessage(models.TransientModel):
+    _inherit = "mail.resend.message"
+
+    @api.model
+    def default_get(self, fields):
+        """Overridden: Add status field for a resend message wizard."""
+        message_id = self._context.get("mail_message_to_resend")
+        mode = self._context.get("mail_status_mode", False)
+        if not message_id or not mode:
+            return super(MailResendMessage, self).default_get(fields)
+        else:
+            rec = super(models.TransientModel, self).default_get(fields)
+            allow_resend = True if mode != "sent_status" else False
+            mail_message_id = self.env["mail.message"].browse(message_id)
+            notification_ids = mail_message_id.notification_ids.filtered(
+                lambda notif: notif.notification_type == "email"
+            )
+            partner_ids = [
+                Command.create(
```

Comment:
This is awesome (for Odoo15 and Odoo16)! No longer need to remember those numbers for creating / modifying many relation fields 😄